ludwig/features/sequence_feature.py

```diff
@@ -141,19 +139,24 @@ class SequenceInputFeature(SequenceFeatureMixin, InputFeature):
     def __init__(self, feature, encoder_obj=None):
         super().__init__(feature)
         self.overwrite_defaults(feature)
+        feature.update(feature['preprocessing'])
```

Comment:
Trying to figure out why this is needed. i guess to update pad_idx?

Reply:
It is essentially used to make the pretrained_model_name_or_path parameter value directly accessible when we initialize the encoder. In the original feature dict, the pretrained_model_name_or_path paramenter is nested inside the key 'preprocessing'  (which is a dict) and is thus not passed to the encoders at initialization time. But, by updating the feature dict to directly contain a key for pretrained_model_name_or_path, the encoder now has access to that value.

Reply:
Got it. This means that if someone specifies it at the encoder level, it gets overridden. For now this is fine, but it's a design choice I have to think about. I may end up going the other way around (the one you specify in the encoder end up used in the preprocessing).

Reply:
Ah gotcha. This makes sense!